test/e2e/apimachinery/watchlist.go

```diff
@@ -52,7 +52,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
-var _ = SIGDescribe("API Streaming (aka. WatchList)", framework.WithFeatureGate(features.WatchList), framework.WithSerial(), func() {
+var _ = SIGDescribe("API Streaming (aka. WatchList)", framework.WithFeatureGate(features.WatchList), func() {
 	f := framework.NewDefaultFramework("watchlist")
 	ginkgo.It("should be requested by informers when WatchListClient is enabled", func(ctx context.Context) {
 		featuregatetesting.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), utilfeature.DefaultFeatureGate, featuregate.Feature(clientfeatures.WatchListClient), true)
```

Comment:
why is it ok to keep setting this clientside feature gate, and drop the WithSerial() marker on the test?

Reply:
It is not okay, I haven't found a serial job that runs the tests, so I temporarily changed the tests (I think there must be a periodic job) just to get a CI signal before merging this PR.

I am going to remove this commit before merging this PR.